pkg/utils/utils.go

```diff
@@ -124,3 +124,15 @@ func GetChrootExtension() string {
 	}
 	return fmt.Sprintf("chroot %s%s", vars.FilesystemRoot, consts.Host)
 }
+
+func GetPciAddressFromInterfaceName(interfaceName string) (string, error) {
+	sysfsPath := filepath.Join(vars.FilesystemRoot, "sys/class/net", interfaceName, "device")
```

Comment:
FilesystemRoot is an empty string by default, I believe you should add slash at the beginning of the path `/sys/class/net`